src/components/buildArguments.ts

```diff
@@ -0,0 +1,100 @@
+import { Uri } from 'vscode';
+import type { Configuration } from '../configuration';
+import type { Message, MessageKind } from './messageBus';
+import { doubleQuotify, singleQuotify } from '../utilities';
+
+export const buildArguments = (
+	configuration: Configuration,
+	message: Omit<
+		Message & { kind: MessageKind.executeCodemodSet },
+		'storageUri'
+	>,
+	storageUri: Uri,
+) => {
+	const args: string[] = [];
+
+	const { command } = message;
+
+	if (command.kind === 'executePiranhaRule') {
+		args.push('-i', singleQuotify(message.targetUri.fsPath));
+		args.push('-c', singleQuotify(command.configurationUri.fsPath));
+		args.push('-o', singleQuotify(storageUri.fsPath));
+
+		// configuration.includePatterns.forEach((includePattern) => {
```

Comment:
I need to figure out why this doesn't work, but this is out of the scope of this PR.